config/projects/chefdk.rb

```diff
@@ -66,7 +68,6 @@
 override :'ruby-windows', version: "2.1.6"
 override :'ruby-windows-devkit', version: "4.7.2-20130224-1151"
 override :'openssl-windows', version: "1.0.1m"
-#override :'ruby-windows', version: "2.0.0-p451"
```

Comment:
Whoops, didn't see the `#` character

![614](https://cloud.githubusercontent.com/assets/261548/8577314/19ae4590-2565-11e5-9b80-624ec5e143a3.gif)